docs/src/Callout.doc.js

```diff
@@ -18,18 +18,20 @@ card(
   <Example
     name="Info Example"
     defaultCode={`
-<Callout
-  type="info"
-  iconAccessibilityLabel="Info icon"
-  title="Your business account was successfully created!"
-  message="Get a badge, show up in more shopping experiences and more. Apply to the Verified Merchant Program—it’s free!"
-  primaryLink={{href: "https://pinterest.com", label:"Get started"}}
-  secondaryLink={{href: "https://pinterest.com", label:"Learn more"}}
-  dismissButton={{
-    accessibilityLabel: 'Dismiss banner',
-    onDismiss: ()=>{},
-  }}
-  />
+  <Box margin={1}>
```

Comment:
@AlbertCarreras could we avoid the wrapping `<Box />`? Ideally this would be easiest copy/paste example and we don't want a 4px margin around each Callout each time

Reply:
Should be good to merge once this comment is addressed

Reply:
Thanks for catching this. I left this dead code that I used for testing 2 Callouts for the demo videos